_studio/mfx_lib/encode_hw/h264/src/mfx_h264_encode_hw_utils_new.cpp

```diff
@@ -2305,6 +2305,15 @@ void MfxHwH264Encode::ConfigureTask(
         task.m_maxPBFrameSize = 0;
     }
 
+    if (video.mfx.RateControlMethod == MFX_RATECONTROL_CQP)
```

Comment:
may be if MFX_RATECONTROL_QVBR?
No sense in max frame size for CQP.

Reply:
The max frame size feature is only enabled in CQP with multipack case.